Bottom: 2e22727463d55e2adb7cbe5b62908a22702d5ec5
Top:    2629d493409407b19f52ec82975d311aa8306042
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 22:56:54 +0000

hid/gtk (GL): I think the polygon renderer works in mask mode now


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 2eb4ec9..e466999 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1386,11 +1386,8 @@ GhidDrawMask (int side, BoxType * screen)
   polygon.BoundingBox = *screen;
   polygon.Flags = NoFlags ();
   SET_FLAG (FULLPOLYFLAG, &polygon);
-  common_fill_pcb_polygon (out->fgGC, &polygon, screen);
+  gui->graphics->fill_pcb_polygon (out->fgGC, &polygon, screen);
   poly_FreeContours (&polygon.NoHoles);
-  /* THE GL fill_pcb_polygon doesn't work whilst masking */
-//  gui->fill_pcb_polygon (out->fgGC, &polygon, screen);
-//  gui->graphics->fill_rect (out->fgGC, 0, 0, PCB->MaxWidth, PCB->MaxHeight);
   ghid_set_alpha_mult (out->fgGC, 1.0);
 
   gui->graphics->use_mask (HID_MASK_OFF);
